scripts/check_frontend_coverage.py

```diff
@@ -285,6 +285,7 @@
     'oppia-short-response-set-input.directive.ts',
     'oppia-short-response-text-input.directive.ts',
     'oppia-visualization-bar-chart.directive.ts',
+    'oppia-visualization-click-hexbins.directive.ts',
```

Comment:
Hi @brianrodri, you should write unit tests for the file to get 100% coverage (in order to avoid regressions)

Reply:
What is the pattern for visualizations? I've seen you're working on PRs to split them up, I'd prefer to follow the final pattern rather than brute-forcing coverage in this PR.

Reply:
Sure, I think we don't have any examples of testing visualizations. What do you think @seanlip?

Reply:
hi @marianazangrossi, sorry I missed this. I have no idea I'm afraid. I think this is up to you and @nithusha21!

Reply:
Please see #9605 for a sample spec.

Reply:
I think it's a great example, can you follow this structure for `oppia-visualization-click-hexbins.directive.ts`?